src/streams.h

```diff
@@ -298,6 +299,30 @@ class CDataStream
     }
 };
 
+class CDataStream : public CBaseDataStream<CSerializeData>
+{
+public:
+    explicit CDataStream(int nTypeIn, int nVersionIn) : CBaseDataStream(nTypeIn, nVersionIn) { }
+
+    CDataStream(const_iterator pbegin, const_iterator pend, int nTypeIn, int nVersionIn) :
+            CBaseDataStream(pbegin, pend, nTypeIn, nVersionIn) { }
+
+#if !defined(_MSC_VER) || _MSC_VER >= 1300
```

Comment:
``_MSC_VER == 1300`` is VC++ 7; no-one's going to be compiling with VC++ earlier than that. Down with conditional compilation cruft!